### PR TITLE
chore: improve the words on org profile and update logos

### LINKED ISSUE #2: Replace the "GreptimeDB" logo with "Greptime" logo
We prioritize the "Greptime" logo as a representation of the whole product lineup, over the "GreptimeDB" one. Currently, the lineup contains at least GreptimeCloud, GreptimeAI, and GreptimeDB.

GreptimeDB brand is associated with a purely open-source project Greptime Inc. produces, and we can make it more clear to collaborate with the proper assumption.

cc @killme2008 this ticket can be linked to the design task and for tracking progress.

### PR DESCRIPTION
**Pull request by tisonkun**
Rendered version - https://github.com/tisonkun/greptime-dot-github/blob/rewording/profile/README.md

This closes #2.

cc @GreptimeTeam/marketing @killme2008

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-## Greptime builds Cloud-Native TSDB Services: Anywhere, Any Scale
-
 <p align="center">
   <picture>
     <source media="(prefers-color-scheme: light)" srcset="/img/logo/text-padding-tined.svg">
```
Comment (killme2008) [inline profile/README.md]: Let's change the logo too.

**Review by killme2008 — APPROVED** [review]

File: profile/README.md (hunk 1 above)
Comment (tisonkun) [reply profile/README.md]: Where is the logo asset, or it's under design?

Comment (killme2008) [reply profile/README.md]: See https://greptime.com/logo-resource

**Comment by killme2008** [comment]
@Wenjie0329 Please take a look.

File: profile/README.md (hunk 1 above)
Comment (tisonkun) [reply profile/README.md]: Updated.

**Review by sunng87 — APPROVED** [review]

**Pull request merged**